doc-design/README.md

```diff
@@ -14,7 +14,7 @@ and a few contributions, which can be installed by running
 ```
 pip3 install --user sphinx
 pip3 install --user sphinx_bootstrap_theme
-pip3 install --user sphinxcontrib.bibtex
+pip3 install --user sphinxcontrib.bibtex==1.0
```

Comment:
@bravache : I don't think that is needed. I compiled with
```
$ python3 -m pip freeze | grep bibtex
bibtexparser==1.1.0
sphinxcontrib-bibtex==0.4.2
```

Reply:
Well, the issue I am trying to avoid with this line is for version > 2.0. I was running into the error described here:
https://github.com/executablebooks/jupyter-book/issues/1137, 
In particular, this comment was what solved it for me: https://github.com/executablebooks/jupyter-book/issues/1137#issuecomment-743993738

Could you check if you can still compile the document with the latest sphinxcontrib.bibtex?